src/scss/_typography.scss

```diff
@@ -13,12 +13,6 @@ h6 {
   }
 }
 
-a,
```

Comment:
Removing these global rules is going to have some unintended consequences (see attached as an example).  Is there something specific you're trying to accomplish by removing these rules that could maybe be added as a component- or template-specific override instead?

![screen shot 2018-08-24 at 11 59 35 am](https://user-images.githubusercontent.com/1649216/44594864-5900f200-a795-11e8-9839-7ff100ec1bc7.png)